File: 2543.py
# Python 3.4.3

def main():
  while True:
    try:
      CONTRA_STRIKE = '0'

      qtd_total_gameplays, meu_identificador = input().split()
      qtd_total_gameplays = int(qtd_total_gameplays)

      qtd_meus_gameplays = 0

      for i in range(qtd_total_gameplays):

        identificador, jogo = input().split()
        if identificador == meu_identificador and jogo == CONTRA_STRIKE:
          qtd_meus_gameplays += 1

      print(qtd_meus_gameplays)

    except EOFError:
      break

if __name__ == '__main__':
	main()

File: 2543.c
#include <stdio.h>

#define CONTRA_STRIKE 0

int main() {
  int qtd_total_gameplays, meu_id, id, jogo, qtd_meus_gameplays;

  while (scanf("%d %d", &qtd_total_gameplays, &meu_id) != EOF) {
    qtd_meus_gameplays = 0;

    for (int i = 0; i < qtd_total_gameplays; i++) {
      scanf("%d %d", &id, &jogo);

      if (id == meu_id && jogo == CONTRA_STRIKE) {
        qtd_meus_gameplays++;
      }
    }

    printf("%d\n", qtd_meus_gameplays);
  }

	return 0;
}